src/kernel/panic.zig

```diff
@@ -281,10 +284,11 @@ pub fn init(mem_profile: *const mem.MemProfile, allocator: *std.mem.Allocator) !
     defer log.logInfo("Done panic\n", .{});
 
     // Exit if we haven't loaded all debug modules
-    if (mem_profile.boot_modules.len < 1)
+    if (mem_profile.boot_modules.len < 1) {
         return;
-    var kmap_start: usize = 0;
-    var kmap_end: usize = 0;
+    }
+    var kmap_start: u32 = 0;
+    var kmap_end: u32 = 0;
```

Comment:
Why change it back to `u32`? Rebase error maybe?

Reply:
ah, maybe, it didn't come up for me when rebasing